Update wikipedia extension (#7359)
raycast/extensions: extensions/wikipedia/CHANGELOG.md

# Wikipedia Changelog

## [Recent Searches] - 2023-07-04

> - Added `Recent Searches` inside the `Search Page` command to veiw a list of your previously viewed articles

Reviewer: ```suggestion
- Added `Recent Searches` inside the `Search Page` to show previously viewed articles
```
Fix typo and simplify sentence

> - Added `Recent Searches` inside the `Search Page` command to veiw a list of your previously viewed articles

Reviewer 2: ```suggestion
- Added `Recent Searches` inside the `Search Page` to show your previously viewed articles
```

Author: can't believe I missed that 😅. thanks for the fix!
